src/main/java/core/basesyntax/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void process(FruitTransaction transaction) {
+        FruitStorage.STORAGE.put(transaction.getFruit(), transaction.getQuantity());
```

Comment:
You should add a DAO layer that is responsible for interaction with DB and use it in your handlers